src/patternfly/components/Expandable/expandable.scss

```
@@ -0,0 +1,38 @@
+.pf-c-expandable {
+  // Expandable Toggle
+  --pf-c-expandable__toggle--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-expandable__toggle--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-expandable__toggle--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-expandable__toggle--PaddingLeft: 0;
+  --pf-c-expandable__toggle--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-expandable__toggle--Color: var(--pf-global--link--Color);
+
+  // Expandable Toggle Icon
+  --pf-c-expandable__toggle-icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-expandable__toggle-icon--Color: var(--pf-global--Color--100);
+
+  // Expandable Expanded
+  --pf-c-expandable__content--MarginTop: var(--pf-global--spacer--xl);
```

Talked to @mceledonia about this and it's too much space. Let's go with 16px instead.

```suggestion
  --pf-c-expandable__content--MarginTop: var(--pf-global--spacer--md);
```